evap/staff/forms.py

```diff
@@ -193,21 +192,22 @@ class EvaluationForm(forms.ModelForm):
         widget=CheckboxSelectMultiple,
         label=_("General questions")
     )
-    course = forms.ModelChoiceField(Course.objects.all(), disabled=True, required=False, widget=forms.HiddenInput())
     last_modified_user_name = forms.CharField(label=_("Last modified by"), disabled=True, required=False)
 
     class Meta:
         model = Evaluation
-        fields = ('name_de', 'name_en', 'is_rewarded', 'is_midterm_evaluation', 'vote_start_datetime', 'vote_end_date',
-                  'participants', 'general_questionnaires', 'last_modified_time', 'last_modified_user_name',
-                  'course')
+        fields = ('course', 'name_de', 'name_en', 'is_rewarded', 'is_midterm_evaluation', 'vote_start_datetime',
+                  'vote_end_date', 'participants', 'general_questionnaires', 'last_modified_time',
+                  'last_modified_user_name')
         localized_fields = ('vote_start_datetime', 'vote_end_date')
         field_classes = {
             'participants': UserModelMultipleChoiceField,
         }
 
     def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+        semester = kwargs.pop('semester', None)
+        super(EvaluationForm, self).__init__(*args, **kwargs)
```

Comment:
why the `(EvaluationForm, self)`?

Reply:
uhm. very good question that i can't answer right now. i don't remember why i put it there - tests seem to be okay when removing it again - so i'll look into it.